pkg/cable/metrics.go

```diff
@@ -84,6 +84,16 @@ var (
 			connectionsStatusLabel,
 		},
 	)
+	privateConnectionsGauge = prometheus.NewGaugeVec(
```

Comment:
```suggestion
	shortConnectionsGauge = prometheus.NewGaugeVec(
```

Should the other vars follow the "private->short" rename too?

Reply:
Yes, will update